src/containers/Board/InfinityScrollBoard/index.tsx

```diff
@@ -4,6 +4,57 @@
  * Copyright (c) 2023 Your Company
  */
 
+import { BoardGrid, GridItem, Wrap } from "./emotion";
+
 export default function InfinityScrollBoard() {
-  return <div>무한스크롤</div>;
+  const dummyData = [
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "1",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "2",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "3",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "4",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "5",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "6",
+    },
+    {
+      title: "test",
+      thumbNail: "www.testimage.co.kr/asdfasdfdsa",
+      postNum: "7",
+    },
+  ];
+  return (
+    <Wrap>
+      <BoardGrid column={3} gridGap={15}>
```

Comment:
```suggestion
      <BoardGrid.ul column={3} gridGap={15}>
```
리스트 형태이기 때문에 ul 태그가 적당한거 같아요

Reply:
반영했습니다.